example/Example.cpp

```diff
@@ -31,6 +31,9 @@
 
 int main(int argc, char* args[])
 { 
+	(void)argc;
+	(void)args;
```

Comment:
Let's rather remove the arguments if they aren't used, instead of marking them as "unused".